src/main/java/uk/gov/hmcts/reform/blobrouter/config/EmailConfig.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.reform.blobrouter.config;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Lazy;
+import uk.gov.hmcts.reform.blobrouter.services.email.MessageSender;
+
+import java.io.File;
+import java.util.Map;
+
+@Configuration
+@Lazy
+public class EmailConfig {
+
+    @Bean
+    @ConditionalOnProperty(name = "spring.mail.host", havingValue = "false")
+    public MessageSender emailSenderDisabled() {
+        return new MessageSender() {
+            @Override
+            public void sendMessageWithAttachments(
+                String subject,
+                String body,
+                String from,
+                String[] recipients,
+                Map<String, File> attachments
+            ) {
+                // do nothing
```

Comment:
Isn't only for dev purposes? If used maybe would be beneficial to log something? What do you think?

Reply:
I agree. added log